specification/maps/data-plane/readme.md

```diff
@@ -45,6 +45,33 @@ directive:
 ``` 
 
 
+### Tag: package-stable-2023-07-01
+
+These settings apply only when `--tag=package-stable-2023-07-01` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2023-07-01'
```

Comment:
Value doesn't match the comment. Also, we typically omit a `-stable` suffix/prefix (only use `-preview` for previews). But if this is following an existing pattern then it's ok.